### PR TITLE
Privacy page

### PR DESCRIPTION
**Pull request by onnovisser**
#501 
Also fixes font loading in development

### CODE REVIEW

File: src/components/PrivacySection.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    email: string
+  }
+}
+
+export function PrivacySection({ title, sections, imprint }: PrivacySectionProps) {
+  return (
+    <CenterContainer as="section">
+      <Stack gap={140} pt={8}>
+        <Stack gap={3}>
+          <Text variant="tag">Imprint</Text>
```
Comment (Hornebom) [inline src/components/PrivacySection.tsx:36]: `as="p || span"`

File: src/components/PrivacySection.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          <Stack gap={1} as="address">
+            <Text variant="heading6">{imprint.name}</Text>
```
Comment (Hornebom) [inline src/components/PrivacySection.tsx:38]: `as="p"`

File: src/components/PrivacySection.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        </Stack>
+        <Stack gap={5}>
+          <Text variant="heading2" as="h1">
+            {title}
+          </Text>
+          <Grid gridTemplateColumns={['1fr', '1fr', '2fr 1fr']}>
+            <Stack gap={8}>
+              {sections.map((section) => (
+                <Stack key={section.title} gap={2}>
+                  <Text variant="heading4" as="h3" role="listitem">
```
Comment (Hornebom) [inline src/components/PrivacySection.tsx:55]: - `as="h2"`
- can only be a `listitem` if the parent is `role="list"`

Comment (onnovisser) [reply src/components/PrivacySection.tsx]: fixed

File: src/components/PrivacySection.tsx (hunk 1 above)
Comment (onnovisser) [reply src/components/PrivacySection.tsx]: made it a `h2` instead

File: src/components/PrivacySection.tsx (hunk 2 above)
Comment (onnovisser) [reply src/components/PrivacySection.tsx]: made it a `h3`, seems more appropriate

**Review by Hornebom — APPROVED** [review]

**Pull request merged**